attacker/attacker.go

```diff
@@ -123,16 +124,19 @@ func RunVegeta(ctx context.Context, requestDicts []map[string]interface{}, testN
 	report := vegeta.NewTextReporter(&metrics)
 	err := report.Report(os.Stdout)
 	if err != nil {
-		zlog.Debug(ctx).Msg("vegeta report command failure")
+		zlog.Info(ctx).Msg("vegeta report command failure")
```

Comment:
Only handle errors once if you can,  so I'd say don't log here with no context, leave it to the calling function to log the error (as it does now).

Reply:
Keeping log level as `Debug` for these type of errors.

Reply:
IMO the log line shouldn't exist, if you want to add more information just format the error message as you do in other places, the error eventually gets logged and it makes a lot more sense to the user. But up to you, it's not part of this PR anymore.